examples/sum_views.cpp

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <beman/any_view/any_view.hpp>
+
```

Comment:
There might be some inspiration in the original impl tests here: https://github.com/huixie90/cpp_papers/tree/main/impl/any_view/test/view

for usage.  I have to say that, as usual, I really dislike the example in the paper -- they're not compelling to me